### PR TITLE
Tidy

### PR DESCRIPTION
**Pull request by KevinJoiner**
Trying to fix CI

This also removes a call the the device definitions registrar that sent a message to a kafka topic for the device-definition-join-processor to use

### CODE REVIEW

File: internal/controllers/geofences_controller.go [hunk 1]
```diff
@@ -275,7 +275,7 @@ func (g *GeofencesController) GetAll(c *fiber.Ctx) error {
 		for _, udtg := range item.R.UserDeviceToGeofences {
 			var deviceDef *ddgrpc.GetDeviceDefinitionItemResponse
 			for _, dd := range dds {
-				if dd.DeviceDefinitionId == udtg.R.UserDevice.DeviceDefinitionID {
+				if dd.Id == udtg.R.UserDevice.DefinitionID {
```
Comment (mateuszpohl) [inline internal/controllers/geofences_controller.go:278]: I did the same fix on my branch this morning :D

Shouldn't we also add an `if` statement on the part below, just in case? In theory `deviceDef` can still be `nil` for some crazy edge cases.

Comment (KevinJoiner) [reply internal/controllers/geofences_controller.go:278]: Yeah I think we can just delete this code soon with privacy zones v2 but James is working on fixing up these test. so I'll let him handle it.
